packages/snap/src/keyring.ts

```diff
@@ -114,6 +116,10 @@ export class AccountAbstractionKeyring implements Keyring {
     return this.#state.config[Number(chainId)]!;
   }
 
+  async getConfigs(): Promise<ChainConfigs> {
```

Comment:
Can you add a JSdoc comment here please so all public methods have documentation :)